jobs/build-current-sqlite.groovy

```diff
@@ -50,25 +38,28 @@ def runScript() {
                           params.GIT_REVISION);
 
    dir("webapp") {
-       sh("make clean_pyc");    // in case some .py files went away
-       sh("make fix_deps");  // force a remake of all deps all the time
-   }
-
-   // We need secrets to talk to gcs, prod.
-   withSecrets() {
-      withEnv(
-         ["CURRENT_SQLITE_BUCKET=${params.CURRENT_SQLITE_BUCKET}",
-          "DEV_APPSERVER_ARGS=${params.DEV_APPSERVER_ARGS}",
-          "SNAPSHOT_NAMES=${params.SNAPSHOT_NAMES}"]) {
-         sh("jenkins-jobs/build_current_sqlite.sh");
-      }
+        // First, we need to get a local webserver running.
+        sh("ssh-agent make start-dev-server-backend WORKING_ON=NONE")
```

Comment:
I think this should be inside the try loop? IIRC, it's possible for the command to fail, but leave containers running? I've only ever noticed it when container startup times out, but still, I know on mac it is possible? Maybe that's an apple oddity that doesn't happen on linux.